core/flyout_separator.ts

```diff
@@ -0,0 +1,17 @@
+import type {IBoundedElement} from './interfaces/i_bounded_element.js';
+import {Rect} from './utils/rect.js';
+
+export class FlyoutSeparator implements IBoundedElement {
```

Comment:
Add license stanza, here and in the other file.

Reply:
Done